common/src/main/java/org/entcore/common/user/UserUtils.java

```diff
@@ -603,6 +603,22 @@ public static void findVisibleManualGroups(final EventBus eb, HttpServerRequest
 		findUsers(eb, request, m, handler);
 	}
 
+	public static Future<JsonArray> areVisible(EventBus eb, String userId, JsonArray checkIds) {
```

Comment:
Plutôt que d'ajouter une nouvelle fonction au service `communication`  qu'il va falloir porter, pourquoi ne pas faire quelque chose comme dans GenericShareService :
```suggestion
	public static Future<JsonArray> areVisible(EventBus eb, String userId, JsonArray checkIds) {
	UserUtils.findVisibles(eb, userId, customReturn,
					extraParams,
					true, true, false,
					"fr", "AND (m.id IN {checkIds}) ",
					users -> {
				promise.complete(users.size() == checkIds.size());
					});
	}
```